test/sanity/validate-modules/schema.py

```diff
@@ -17,52 +17,62 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from voluptuous import PREVENT_EXTRA, Any, Required, Schema
+from ansible.compat.six import string_types
```

Comment:
ansible.compat.six has gone away in devel.  Now you can use ansible.module_utils.six everywhere.